src/main/kotlin/build/buf/intellij/index/BufIndexes.kt

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022-2024 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build.buf.intellij.index
+
+import build.buf.intellij.module.ModuleKey
+import com.intellij.openapi.project.Project
+import com.intellij.psi.search.GlobalSearchScope
+import com.intellij.util.indexing.FileBasedIndex
+import com.intellij.util.indexing.ID
+
+/**
+ * Contains utility methods to retrieve data from `buf.yaml` and `buf.lock` indexes.
+ */
+object BufIndexes {
```

Comment:
There were compiler warnings about putting anything other than a logger in the companion object for indexes, so I extracted the index ID and helper methods out to a separate object.